sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jdbc/JdbcPropertiesBeanPostProcessor.java

```diff
@@ -46,14 +48,18 @@ class JdbcPropertiesBeanPostProcessor implements BeanPostProcessor, EnvironmentA
 
     private GenericApplicationContext applicationContext;
     private Environment environment;
+    private final AzureGlobalProperties azureGlobalProperties;
+
+    JdbcPropertiesBeanPostProcessor(AzureGlobalProperties azureGlobalProperties) {
```

Comment:
Can we get the `AzureGlobalProperties` bean from the `applicationContext` instead of passing it into the ctor?